### PR TITLE
Cleanup Documentation of Herman styles and patterns

### PR DESCRIPTION
**Pull request by mirisuzanne**
Not the documentation of Herman's API, but the self-documentation of Herman's styles.

### CODE REVIEW

**Comment by mirisuzanne** [comment]
@jgerigmeyer I'm not sure what I touched to break this test. Take a look when you have a chance?

File: scss/config/_scale.scss [hunk 1]
```diff
@@ -22,83 +22,125 @@ $text-ratios: (
...
 $pattern-sizes: (
-  'border': 8px,
-  'page': 50rem,
+  'nav-underline': 4px,
+  'icon': 28px,
 
-  'page-break': 50em,
-  'nav-small': 21em,
-  'nav-medium': 32em,
-  'nav-large': 65em,
```
Comment (jgerigmeyer) [inline scss/config/_scale.scss]: @mirisuzanne This was the breaking change. The JS was using `pattern-sizes - nav-large` as the breakpoint... now it's `layout-sizes - nav-break`.

**Comment by mirisuzanne** [comment]
@stacyk @jgerigmeyer @SondraE - this project is not "done" but I think we can merge what we have for the launch. I'd love to get your review, and merge still today.

File: scss/config/_scale.scss [hunk 2]
```diff
@@ -22,83 +22,125 @@ $text-ratios: (
...
+/// for the root of our typography,
+/// with a `px`-based fallback for older browsers
+/// and unit conversions.
+/// We also provide `small` and `large`
+/// font-sizes to be used in special cases.
+///
+/// These sizes should only be accessed for establishing
+/// the root size on the `html` tag,
+/// or assigned more semantic names before applying to
+/// patterns and compoents.
```
Comment (jgerigmeyer) [inline scss/config/_scale.scss]: "components"

File: scss/initial/_icons.scss [hunk 3]
```diff
@@ -3,25 +3,55 @@
...
+/// - `size=none` :: {`string` | `none`}<br/>
+///   Currently supported icon sizes include
+///   `'nav-icon'` and `'footer-logo'`.
+/// - `class=none` :: {`string` | `none`}<br/>
+///   Any classes that should be applied to the icon.
+///
 /// @example njk
 ///   {% import 'utility.macros.njk' as utility %}
-///   {{ utility.icon('menu', size='icon-small') }}
+///   {{ utility.icon('menu', size='nav-icon') }}
```
Comment (jgerigmeyer) [inline scss/initial/_icons.scss]: This block is not being displayed. Should it be included in `/// @group style-icons`?

File: scss/layout/_regions.scss [hunk 4]
```diff
@@ -1,16 +1,27 @@
 // Layout Regions
 // ==============
-
-
-[data-sassdoc-region] {
+/// # Herman Layout Regions
+/// The Herman layout is comprised of several "regions" --
+/// such as the banner, nav, and main areas --
+/// as well as the `app` and `container` wrapping regions.
```
Comment (jgerigmeyer) [inline scss/layout/_regions.scss]: Again, a block without a `@group`. I assume `@group style-regions`?

File: scss/previews/_font.scss [hunk 5]
```diff
@@ -2,17 +2,35 @@
...
-  font-size: size('small');
+.font-specimen {
+  display: flex;
+  flex-wrap: wrap;
+  margin: size('gutter') negative('shim') 0;
+
+  @supports (display: grid) {
+    display: grid;
+    grid-auto-rows: auto;
+    grid-gap: size('gutter'); // sass-lint:disable-line no-misspelled-properties
```
Comment (jgerigmeyer) [inline scss/previews/_font.scss]: This shouldn't be necessary (the `sass-lint:disable-line`). Have you run `yarn` recently?

File: scss/samples/_sizes-ratios.scss [hunk 6]
```diff
@@ -128,14 +130,22 @@ $demo-sizes: (
...
 /// ```scss
 /// /// @sizes demo-sizes {ruler}
 /// ```
 /// @sizes demo-sizes {ruler}
 /// @group demo_sizes
 
 
+/// ### Large Rulers
+/// ```scss
+/// /// @sizes demo-sizes {ruler}
```
Comment (jgerigmeyer) [inline scss/samples/_sizes-ratios.scss]: `ruler-large`

File: scss/layout/_regions.scss (hunk 4 above)
Comment (mirisuzanne) [reply scss/layout/_regions.scss]: Yeah.

**Comment by mirisuzanne** [comment]
@jgerigmeyer should be fixed, if you want to merge/release tonight or in the morning.

**Pull request merged**